src/Services/ArtistDisplayService.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+class ArtistDisplayService {
```

Comment:
It looks like you made this, and then didn't use it. Can you move the artist display code from index into this please?

Reply:
Once you've done that, this will need unit tests as well please :)